tests/linux_packages/maven_test.py

```diff
@@ -16,7 +16,9 @@
 
 import unittest
 import mock
+import posixpath
```

Comment:
import posixpath
import unittest

import mock
from perfkitbenchmarker import flags
from perfkitbenchmarker.linux_packages import INSTALL_DIR
from perfkitbenchmarker.linux_packages import maven
from perfkitbenchmarker.tests import pkb_common_test_case

Reply:
This
```
from perfkitbenchmarker.tests import pkb_common_test_case
```
will give me the error
```
ModuleNotFoundError: No module named 'perfkitbenchmarker.tests'
```
because the tests are not under perfkitbenchmarker, but in the root dir of the repo.

Also, why is mock not part of the first import block? Like:
```
import mock
import posixpath
import unittest

from perfkitbenchmarker import flags
from perfkitbenchmarker.linux_packages import INSTALL_DIR
from perfkitbenchmarker.linux_packages import maven
from tests import pkb_common_test_case
```

Reply:
@yuyantingzero I am guessing you are using `flake8-import-order` for internal tests, which explains the ordering issues in my PR. Since I am not aware of the ordering rules you use, I made the exact changes you requested, except the last module that was causing the import error. Hope it works now.